test/e2e/specs/test.spec.js

```diff
@@ -1,52 +1,177 @@
 /* eslint-disable ui-testing/no-disabled-tests */
 describe('Vaults UI Test Cases', () => {
   context('Test commands', () => {
-    let collateralizationRatio;
-    it(`should setup Keplr account and connect with Agoric Chain`, () => {
-      cy.origin('https://wallet.agoric.app/', () => {
-        cy.visit('/');
-      });
-      cy.acceptAccess().then(taskCompleted => {
-        expect(taskCompleted).to.be.true;
-      });
+    const phrasesList = {
+      emerynet: {
+        interNetwork: 'Agoric Emerynet',
+        isLocal: false,
+      },
+      local: {
+        interNetwork: 'Local Network',
+        isLocal: true,
+      },
+    };
+    const networkPhrases = phrasesList[Cypress.env('AGORIC_NET') || 'local'];
 
-      cy.origin('https://wallet.agoric.app/', () => {
-        cy.visit('/wallet/');
+    it('should setup the wallet', () => {
+      if (networkPhrases.isLocal) {
+        cy.setupWallet();
+      } else {
+        const walletAddress = {
+          value: null,
+        };
+        cy.setupWallet({
+          createNewWallet: true,
+          walletName: 'my created wallet',
+        });
+        cy.origin('https://wallet.agoric.app/', () => {
+          cy.visit('/wallet/');
 
-        cy.get('input.PrivateSwitchBase-input').click();
-        cy.contains('Proceed').click();
+          cy.get('input.PrivateSwitchBase-input').click();
```

Comment:
this class name may change. please select by something stable in the UI. e.g. something the user sees